apps/tailwind-components/components/Story.vue

```diff
@@ -5,8 +5,13 @@ defineProps<{
 }>();
 </script>
 <template>
-  <h1>{{ title }}</h1>
-  <p v-if="description">{{ description }}</p>
-  <hr />
-  <slot></slot>
+  <div>
+    <h1 class="text-heading-6xl text-favorite hover:text-favorite-hover">
+      {{ title }}
+    </h1>
+    <p class="mt-2" v-if="description">{{ description }}</p>
+    <slot></slot>
+    <div class="mt-4">Source code:</div>
```

Comment:
This should be an h2 element rather than a div. We could also move this into a collapsible section so that it can be hidden/shown.

Reply:
yes, do we have collapse component yet?

Reply:
Not yet. It could be added to the forms story as it could be useful in multiple places.